src/ragapp/config/agents.yaml

```
@@ -0,0 +1,27 @@
+8d53880d-906d-404c-b5f4-91fb07567b85:
+  created_at: 2024-09-16 02:53:57.746566
+  name: Default
+  role: Use the provided knowledge to answer user question
```

similar to crewai: a role starts with what the "person" is, e.g. a researcher
```suggestion
  role: Expert in answering questions using the provided knowledge
```